openpype/hosts/maya/api/lib.py

```diff
@@ -2068,8 +2073,12 @@ def get_frame_range():
 
     # Set frame start/end
     project_name = legacy_io.active_project()
+    task_name = legacy_io.Session["AVALON_TASK"]
     asset_name = legacy_io.Session["AVALON_ASSET"]
```

Comment:
Please be aware that we are trying to move from the use of `legacy_io`. This was already changed in `release/next-minor` branch like this:

```suggestion
     # Set frame start/end
    project_name = get_current_project_name()
    task_name = get_current_task_name()
    asset_name = get_current_asset_name()
```